proto/ibc/applications/ibc_query/v1/tx.proto

```diff
@@ -54,8 +54,9 @@ message MsgSubmitCrossChainQueryResult {
   QueryResult result = 3;
   bytes data = 4;
   string sender = 5;
+  string query_sender = 6;
```

Comment:
1. Does sender field mean relayer and query_sender mean query requester?

2. query_sender -> querySender

Reply:
1. yes

Reply:
2: sorry my mistake..